src/InfinitudePlatform.js

```diff
@@ -43,6 +49,15 @@ module.exports = class InfinitudePlatform {
       }.bind(this)
     );
   }
+	
+ configureAccessory(accessory) {
+    this.initializeZones(false).then(
+      function() {
+        this.accessories[accessory.UUID] = accessory;
+        this.configureSensorAccessory(accessory);
+      }.bind(this)
+    );
+  }
```

Comment:
I don't think you can dupliate `configureAccessory` like this. You'll have to alter the function that's already there.